benchmarks/query_benchmarks/query_values_list/benchmark.py

```diff
@@ -0,0 +1,13 @@
+from ...utils import bench_setup
+from .models import Book
+
+
+class QueryValuesList:
+    def setup(self):
+        bench_setup(migrate=True)
+
+    def teardown(self):
+        Book.objects.all().delete()
+
+    def time_query_values_list(self):
+        list(Book.objects.values_list("title"))
```

Comment:
This benchmark is rather quick (0.000189seconds). Can we repeat it so we it (10x say) to give a longer duration, i fear with it being so quick it will become subject to noise.  

See https://github.com/smithdc1/django-asv/runs/6720503551?check_suite_focus=true#step:5:71